src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,141 @@
+import React, {Component, createRef} from 'react';
+import uniqId from 'uniqid';
+import {Button} from '../Button/Button';
+import ErrorImg from '../../assets/error.png'
+
+import './AddUserForm.scss';
+
+const CN = 'user-form';
+
+class AddUserForm extends Component {
+    constructor(props) {
+        super(props);
+        // this.userNameRef = createRef(); // аналогично записи в строке 19
+        // this.userLastNameRef = createRef();
+    }
+
+    // todo: добавить state, в него добавить пропертю, которая будет отображать есть ли ошибка или нет.
+    //  допустим это будет строка currentError
+    //  по умолчанию она будет пустой
+    //  если произошла ошибка - в эту пропертю запишется строка, какая именно ошибка произошла
+    state = {
+        currentError: {
+            userName: '',
+            userLastName: ''
+        }
+    };
+    userNameRef = createRef();
+    userLastNameRef = createRef();
+
+    onInputChange = (e) => {
+        console.log(e.target.value)
+        // todo: если в стейте была записана какая-то ошибка - зачищаем ее при изменении инпута
+        //  поскольку в ситуации если пользователь ничего - не ввел - увидел ошибку - понял, что не так - начал вводить данные
+        //  было бы хорошо не пугать его висящим сообщением, что все таки что-то не так
+        this.setState({currentError: ''});
+    };
+
+    onSubmit = (e) => {
+        e.preventDefault();
+
+        // Todo: добавить здесь проверку, если пользователь НЕ ВВЕЛ в инпуты ничего - не создаем пост
+        //  и обновляем currentError в стейте, кладем сообщение, что пошло не так
+        //  если пользователь ввел данные - создаем пост
+
+        if (this.userNameRef.current.value.trim() === "") {
```

Comment:
You can create 2 separate variables for inputs values
for example
````
const userNameValue = this.userNameRef.current.value;
const userLastNameValue = this.userLastNameRef.current.value;
````
and then use inside if conditions